app/controllers/groups_controller.rb

```diff
@@ -122,6 +122,7 @@ def send_deny_access_message(user, group)
       recipient: user,
       text: t('groups.send_deny_access_message.message', user: current_user.name, group: group.name),
```

Comment:
Previously I set the locale to English here to make sure the message was in English in the database. This did not work however. I think Rails called the text getter of Message before writing the data to the database. Since the message gets translated in the text getter it would then not necessarily be in English in the database. I thought about adding a parameter "translate: false" to the getter but I think it is not that important to have the messages in a unified language in the database since they always get translated anyways.

Reply:
Wouldn't it be possible to store the I18n key and the hash for parameters (if required) to the database? Or, just store some form of enum, so that we have a mapping from the enum to the locale key separately? 

I think, always storing the English locale isn't the way to go, then I would rather use the "real" translation. However, just some abstract identifier might be even better. The idea behind the enum would be that we can still change the translations and their keys, without invalidating the database entries (because the further indirection through the mapping can be updated in code more easily).

Reply:
I think we don't need to worry about that because currently the text from the database gets overwritten in Message#text anyways. An end user never sees the text from the database. We could also leave the text column empty for group messages.

Reply:
Alright, thanks, that makes sense. And with the `param_type`, we already have the indirection I thought of.

One final question on this topic: Wouldn't it then make sense to prevent storing the translation at all, if we never use it? Then, we would just set the `text` column to `NULL` on those occasions (in the collections and groups controller). 

By the way, in the groups controller, we can reduce the number of "redundant" message methods by passing the `param_type` as an argument. For the `Message` model, we can modify the presence validator for `text` by appending `, unless: -> { %w[group group_accepted group_declined collection].include?(param_type) }`.